DistTasks.yml

```diff
@@ -19,7 +19,7 @@ version: "3"
 
 vars:
   CONTAINER: "docker.elastic.co/beats-dev/golang-crossbuild"
-  GO_VERSION: "1.17.8"
+  GO_VERSION: "1.19"
```

Comment:
Currently, the latest go release is `1.19.4`, I think the containers require an exact version.

Reply:
crossbuild images don't have a `1.19.4` tag yet, but updated to `1.19.3`